spec/embedded_sass.proto

```diff
@@ -1060,3 +1067,13 @@ enum CalculationOperator {
   // The division operator.
   DIVIDE = 3;
 }
+
+// The built-in Node.js Package Importer, which is a Package Importer that
+// resolves using the standards and conventions of the Node.js ecosystem. It
+// enables a `pkg:` URL scheme for usage with `@use` that directs an
+// implementation to resolve a URL within a dependency. It must be instantiated
+// with an`entry_point_url`, which is passed as the `parentURL` to the Node.js
+// resolution algorithm.
```

Comment:
The last sentence should probably be moved to the documentation of `entry_point_url`. Also, we should more clearly specify what a valid URL is (does it have to be absolute? does it have to be `file:`? if it is relative, to what?)

Reply:
Moved and documented in daf424181f903bfadeb37fcce494ea82623d298d